### ISSUE TITLE
Add support for multiple project IDs to the Gradle plugin

### ISSUE THREAD

**Issue by aweigold** [Kentico/cloud-generators-java#9]
A new extension to the plugin should be added for users who want to make generate models in multiple projects.  The extension should look something like this:

'''
kenticoModels {
  proj1 = kenticoModel{}
  proj2 = kenticoModel{}
}
'''

**Comment by mclay** [comment]
Hey, I would like to work on this. Could you please elaborate a bit on the expected behavior respectively use case for this? 
@aweigold @petrsvihlik

**Comment by aweigold** [comment]
Yes.  Currently the plugin will generate source code based off the content type api Kentico Cloud provides.  Within Kentico Cloud, you can create multiple projects associated with a project ID, which will have it's own set of content types defined.  The current iteration of the plugin allows configuration of one single project, however many consumers of this may want to leverage the same build file for multiple projects.  The expected behavior would be an additional extension or container in the plugin to allow specification of multiple project ids, package names, and output directories.
